bedrock/settings/base.py

```diff
@@ -596,93 +596,587 @@ def facebook_tab_url_lazy():
-    'es-CL': DONATE_SPANISH_LINK,
-    'es-ES': DONATE_SPANISH_LINK,
-    'es-MX': DONATE_SPANISH_LINK,
-    'fr': (
-        'https://donate.mozilla.org/fr/?presets=100,50,25,15&amount=50'
-        '&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org'
-        '&utm_medium=referral&utm_content={source}&currency=eur'
-    ),
-    'he': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-ils/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_ILS'
-    ),
-    'hu': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-huf/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_HUF'
-    ),
-    'ja': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-jpy/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_JPY'
-    ),
-    'nb-NO': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-nok/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_NOK'
-    ),
-    'nn-NO': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-nok/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_NOK'
-    ),
-    'pl': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-pln/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_PLN'
-    ),
-    'pt-BR': (
-        'https://donate.mozilla.org/pt-BR/?presets=375,187,90,55&amount=187'
-        '&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org'
-        '&utm_medium=referral&utm_content={source}&currency=brl'
-    ),
-    'ru': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-rub/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_RUB'
-    ),
-    'sv-SE': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-sek/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_SEK'
-    ),
-    'th': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-thb/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_THB'
-    ),
+DONATE_LINK = ('https://donate.mozilla.org/{locale}/?presets={presets}'
+    '&amount={default}&ref=EOYFR2015&utm_campaign=EOYFR2015'
+    '&utm_source=mozilla.org&utm_medium=referral&utm_content={source}'
+    '&currency={currency}')
+
+DONATE_PARAMS = {
```

Comment:
Please double check the list of locales with the locales actually supported on Bedrock, no point adding locales that will never be available on mozilla.org. Where did you get the current list?

From a very quick check (there might be others):
* af-ZA -> af
* au (not even a locale, it's a region, and we don't support en-AU anyway)
* ca-ES -> ca
* ch (again, not a locale)
* en-CA
* en-NZ
* en-PH
* et-EE -> et
* eu-ES
* fa-IR
* fl and fl-FL. Not even sure what this is… Did you mean Finnish (fi)?
* fr-FR, fr-CA
* fy
* hr-HR
* hu-HU
* id-ID
* ig-NG
* is-IS
* no
* xh-ZA